js/viz/docs/doccharts.js

```diff
@@ -1623,6 +1659,34 @@ var dxChart = {
             text: undefined
         },
         /**
+        * @name dxchartoptions_argumentaxis_workdaysonly
+        * @publicName workdaysOnly
+        * @type boolean
+        * @default false
+        */
+        workdaysOnly: false,
```

Comment:
Check case for this word too.

Reply:
This word doesn't appear anywhere in the DevExtreme.